pkgs/desktops/pantheon/desktop/elementary-session-settings/default.nix

```diff
@@ -80,9 +81,20 @@ stdenv.mkDerivation rec {
     sha256 = "0n1m41aapr58rb1ffvfkjq6c6w3f0ynjzzhja50s4di98p4m7y0q";
   };
 
+  patches = [
+    # Map Pantheon required components by g-s-d versions
+    # https://github.com/elementary/session-settings/pull/23
+    (fetchpatch {
+      url = "https://github.com/elementary/session-settings/commit/39918f4ec64fa9ed5affa109d6a692b97ae4ff01.patch";
+      sha256 = "0v2kqcsibymnslnnw4v67yh098znsrhrcycgxkw8vymvwlinc502";
+    })
+  ];
+
   nativeBuildInputs = [
+    elementary-settings-daemon
```

Comment:
??

Reply:
my patch needs it https://github.com/elementary/session-settings/commit/39918f4ec64fa9ed5affa109d6a692b97ae4ff01.patch. version compare with the info from pkgconfig.
I consider that native

Reply:
We are asking what version will be installed so I would still consider it a run-time dependency – though in this case either will work.